### ISSUE TITLE
About  visualization of the NNGuide

### ISSUE THREAD

**Issue by 0nem0relight** [roomo7time/nnguide#3]
![image](https://github.com/roomo7time/nnguide/assets/140148974/bb173566-b102-4503-af14-d64b57b488d9)
Great works!But i want to know How is this graph drawn, with which data set?

**Comment by roomo7time** [comment]
Thank you for your interest! We have used `sklearn.datasets.make_moons` for the dataset. We trained a single hidden layer MLP on the dataset with the Adam optimizer. We have used the softmax classifier confidence due to its limited output range and hence easier calibration in the visualization.
